packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -769,6 +769,22 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         return (currentActiveRequests() && getVotePhase() == Phase.Reveal);
     }
 
+    function _computePendingStakes(address wallet, uint256 amount) internal override {
```

Comment:
Could we store this per round rather than per vote?

Reply:
Totally! Very good observation, this simplifies things and improves gas consumption even more. 
Updated: [here](https://github.com/UMAprotocol/protocol/pull/4168/commits/12b51bee2b1f44e974c18393f559aac328c5700c)